bot/monkey_patches.py

```diff
@@ -50,3 +52,36 @@ async def honeybadger_type(self, channel_id: int) -> None:  # noqa: ANN001
             pass
 
     http.HTTPClient.send_typing = honeybadger_type
+
+
+def patch_message_converter() -> None:
```

Comment:
nit: this is only PartialMessageConvertor

Reply:
nvm, found the relevant line:

https://github.com/Rapptz/discord.py/blob/1a4e73d59932cdbe7bf2c281f25e32529fc7ae1f/discord/ext/commands/converter.py#L386